elife/php7.sls

```diff
@@ -46,6 +46,7 @@ php-log:
         - name: /var/log/php_errors.log
         - user: {{ pillar.elife.webserver.username }}
         - mode: 640
+        - replace: False
```

Comment:
Diving back to the commit message, it seems that they want `replace` to be explicit when `source` is not used:
```If set to False and the file already exists, the file
will not be modified even if changes would otherwise be made.
Permissions and ownership will still be enforced, however.```

```
[WARNING ] State for file: /var/log/php_errors.log - Neither 'source'
nor 'contents' nor 'contents_pillar' nor 'contents_grains' was defined,
yet 'replace' was set to 'True'. As there is no source to replace the
file with, 'replace' has been set to 'False' to avoid reading the file
unnecessarily.
```